src/main/java/com/alkemy/ong/service/OrganizationService.java

```diff
@@ -28,4 +33,15 @@ public OrganizationDto getPublicOrganization(String organizationId) {
                     "No se encontró organización con esos datos");
         }
```

Comment:
The comment in the Update method is in English and this one in Spanish for a reason?

Reply:
The first comment does not correspond to my code, that's why the difference.